src/content/interop/c-interop.md

```diff
@@ -257,30 +257,30 @@ to write the Dart bindings that integrate with the C code.
 To have Dart create FFI wrappers from C header files,
 use the [`package:ffigen`][ffigen] binding generator.
 
-## Build and bundle native assets {:#native-assets}
+## Dart packages containing native code {:#native-assets}
 
 :::note
-The native assets are **experimental**,
-and [in active development]({{site.repo.dart.sdk}}/issues/50565).
+Support for build hooks is in **preview** and
+can be used on the `main` channel.
 :::
 
-The _Native Assets_ feature should resolve a number of issues associated with
-the distribution of Dart packages that depend on native code.
-It does so by providing uniform hooks for integrating with various
-build systems involved in building Flutter and standalone Dart applications.
+Dart _build hooks_ (formerly known as _native assets_) enable
```

Comment:
I am worried about us switching terminology here. It makes the sections that follow not make as much sense. If this change was a personal preference, let's stick with the original terminology.

If this change is tied to code changes that update the terminology, you should also look at the subsections and update them as needed to match.